src/api/custom-resolvers/queries/get-countries-and-currencies/index.ts

```diff
@@ -0,0 +1,39 @@
+import getCountries from '../../../helpers/get-countries';
+import apimCurrencies from '../../../helpers/get-APIM-currencies';
+import { Context } from '../../../types';
+
+/**
+ * getCountriesAndCurrencies
+ * - Get countries from our DB.
+ * - Get and map currencies from APIM.
+ * @param {Object} root: GraphQL root variables
+ * @param {Object} variables: GraphQL variables for the getCountriesAndCurrencies query
+ * @param {Context} context: KeystoneJS context API
+ * @returns {Promise<Object>} APIM countries and currencies
+ */
+const getCountriesAndCurrencies = async (root: any, variables: object, context: Context) => {
+  try {
+    console.info('Getting countries and currencies (getCountriesAndCurrencies helper)');
+
+    const [countries, currenciesResponse] = await Promise.all([await getCountries(context), await apimCurrencies.get()]);
+
+    if (!currenciesResponse.success) {
```

Comment:
do we not need a check for if no countries as well?

Reply:
No - `getCountries` is an internal async call which just calls our DB. It does not return `success: true`, it returns an array. If this fails, an error will be thrown and this new function/resolver will catch it.

The other function `apimCurrencies.get`, is an external async call which _does_ return `success: true`.